src/functionalTest/java/uk/gov/hmcts/reform/draftstore/endpoint/v3/Create.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.draftstore.endpoint.v3;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import uk.gov.hmcts.reform.draftstore.data.DraftStoreDAO;
+import uk.gov.hmcts.reform.draftstore.service.UserIdentificationService;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(DraftController.class)
+public class Create {
```

Comment:
Should name it CreateTest or sth.

Reply:
Maybe it would be worth checking side effects, i.e. make sure dao.create() has or has not been called.